### ISSUE TITLE
question about  the data used to construct H

### ISSUE THREAD

**Issue by xiangz201** [iMoonLab/HGNN#11]
Hi.It seems you construct hypgraph incidence matrix H from both training data and testing data.

In the function hypergraph_utils.construct_H_with_KNN(),the data pts that fed in seems has both training data(idx_train) and testing data(idx_test).

If we use both  training data and testing data when we train the model, then use the same testing data to evaluate the model,  is the result still accurate?

**Comment by yifanfeng97** [comment]
Yes, our process is ok. Because in this paper we use transductive learning scheme to evaluate our HGNN. A classical hypergraph transductive learning builds a hypergraph that contains all vertices in the training and validation set.  You can refer to the definition of graph/hypergraph transductive learning for detail.
